packages/extensions-sdk/src/cli/commands/validators/check-built-code.ts

```diff
@@ -0,0 +1,53 @@
+import { EXTENSION_PKG_KEY } from '@directus/extensions';
+import fse from 'fs-extra';
+import { type Ora } from 'ora';
+import type { Report } from '../../types.js';
+
+const checkBuiltCode = {
+	name: 'built-code',
+	handler: async (spinner: Ora, reports: Array<Report>) => {
+		spinner.text = 'Check for built code';
+
+		let codePath = '/dist';
+
+		if (await fse.pathExists(`${process.cwd()}/package.json`)) {
+			const packageFile = await fse.readJson(`${process.cwd()}/package.json`);
+
+			if (packageFile[EXTENSION_PKG_KEY]) {
+				const { path } = packageFile[EXTENSION_PKG_KEY];
+				const message = `Path ${path} found in ${EXTENSION_PKG_KEY}`;
```

Comment:
`path` can be `undefined` if it isn't set. We would need to check this and can't assume it's always valid.